app/code/core/Mage/Api/Model/Server/Handler/Abstract.php

```diff
@@ -35,9 +35,9 @@ public function __construct()
      * @param string $errorFile
      * @return bool
      */
-    public function handlePhpError($errorCode, $errorMessage, $errorFile)
+    public function handlePhpError($errorCode, $errorMessage, $errorFile, $errLine)
     {
-        Mage::log($errorMessage . $errorFile);
+        Mage::log($errorMessage . '  in ' . $errorFile . ' on line ' . $errLine);
```

Comment:
I like it. We could also set the level to `Zend_Log::ERR` cause it appears that it's logging them as `DEBUG` messages.

Reply:
Also there is an extra space in `'  in '`.